lib/stream-chat/client.rb

```diff
@@ -36,13 +37,15 @@ class Client
     # @example initialized the client with a timeout setting
     #   StreamChat::Client.new('my_key', 'my_secret', 3.0)
     #
-    def initialize(api_key = '', api_secret = '', timeout = 6.0, **options)
-      @api_key = api_key || ENV['STREAM_KEY']
-      @api_secret = api_secret || ENV['STREAM_SECRET']
-      @timeout = timeout || ENV['STREAM_CHAT_TIMEOUT']
+    def initialize(api_key, api_secret, timeout = nil, **options)
+      raise ArgumentError, 'api_key and api_secret are required' if api_key.to_s.empty? || api_secret.to_s.empty?
+
+      @api_key = api_key
+      @api_secret = api_secret
+      @timeout = timeout || DEFAULT_TIMEOUT
       @options = options
       @auth_token = JWT.encode({ server: true }, @api_secret, 'HS256')
-      @base_url = options[:base_url] || ENV['STREAM_CHAT_URL'] || BASE_URL
+      @base_url = options[:base_url] || DEFAULT_BASE_URL
```

Comment:
this is breaking if somebody is using from env directly (very unlikely though), let's mention in changelog clearly

Reply:
yup, this is a pretty new feature so it's unlikely, but I just added it to the body of the commit message. I think the changelog will reflect it. if not, I'll add it manually there.